backend/views/test_instructions_view.py

```diff
@@ -0,0 +1,19 @@
+from rest_framework.views import APIView
+from rest_framework import permissions
+from views.retrieve_test_view import TEST_INSTRUCTIONS, retrieve_test_data, is_test_public
+
+
+class TestInstructionsSet(APIView):

```

Comment:
I thought we weren't doing the instructions yet?

Reply:
we aren't, but the API is very similar, so I added it while I was doing the others.